x/marker/client/cli/cli_test.go

```diff
@@ -721,22 +721,23 @@ func (s *IntegrationTestSuite) TestMarkerTxCommands() {
 			false, &sdk.TxResponse{}, 0,
 		},
 		{
-			"finalize",
-			markercli.GetCmdFinalize(),
+			"mint supply",
```

Comment:
It'd be good to add a second `mint` test case here that has a recipient (one with a recipient, and one without).